config/locales/en.yml

```
@@ -203,10 +203,10 @@ en:
             explorer_vote: View at Explorer Vote
             message: The election has been created. We're waiting for the start date
               time to arrive.
-            preview: Preview landing page
             secret_until_the_end: The election results will be secret until the end
               of the voting period.
             title: Election created
+            view: Preview landing page
```

This should change to: 
```suggestion
            view: View landing page
```

---

indeed, thanks!